### DISCUSSION TITLE
大积分任务修复进度（基于develop分支）

### DISCUSSION THREAD

**Discussion by Polaris-cn10086** [RayWangQvQ/BiliBiliToolPro#683] [General]
**当前有问题的任务（基于develop分支）：**
- [x] 浏览装扮商城主页
- [ ] 观看任意正片内容

两者都是由于api更新导致的出错，其中`观看任意正片内容`任务由于B站更新，现在只能由大陆手机版B站才能完成任务，任务名称从`ogvwatch`变更为`ogvwatchnew`

如果还有其他问题，欢迎在下面提出，我会尽可能尝试修复

**Comment by Polaris-cn10086** [comment]
目前`浏览装扮商城主页`功能修复完成，`观看剧集内容`功能能够领取，但完成的api还在分析。

![Screenshot_2024-04-14-09-18-02-599_tv danmaku bili](https://github.com/RayWangQvQ/BiliBiliToolPro/assets/63545780/569880de-40a0-4623-a331-3d17565c995b)

**Comment by RayWangQvQ** [comment]
赞

**Comment by ypq123456789** [comment]
修复的如何了大佬

**Comment by YK19971224** [comment]
这个还在修复吗
